### PR TITLE
Tesseron ASM Ticket

### PR DESCRIPTION
**Pull request by developmentLL**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/Tesseron ASM Ticket/readme.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+## Tesseron ASM Ticket Connector
+With Tesseron ASM, we offer your company an innovative solution for cooperation with your customers, partners and suppliers.This connector allows you to use Tesseron ASM Ticket API to create, edit and search Tickets.
+
+
```
Comment (v-EgorKozhadei) [inline certified-connectors/Tesseron ASM Ticket/readme.md:4]: @developmentLL some of the required headings for readme.md file are missing. Please refer to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1).

File: certified-connectors/Tesseron ASM Ticket/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    "policyTemplateInstances": [
+      {
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "https://@connectionParameters('instanceURL')/ApiTicket"
+        },
+        "templateId": "dynamichosturl",
+        "title": "instanceURL"
+      }
+    ],
+    "publisher": "luithle + Luithle GmbH"
```
Comment (v-EgorKozhadei) [inline certified-connectors/Tesseron ASM Ticket/apiProperties.json:40]: @developmentLL the stackowner property is missing. Please refer to [apiProperties.json template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json).

File: certified-connectors/Tesseron ASM Ticket/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+          "tooltip": "SProvide your Tesseron ASM API key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#218472",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Tesseron ASM Ticket/apiProperties.json:29]: @developmentLL make sure that your icon background color is not transparent and it matches the color from this property. I noticed that you have submitted an icon along with other files, it has transparent background. It's not allowed.

File: certified-connectors/Tesseron ASM Ticket/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,819 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Tesseron ASM Ticket",
+    "description": "This connector allows you to use Tesseron ASM Ticket API to create, edit and search Tickets.",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Tesseron ASM Ticket/apiDefinition.swagger.json:5]: @developmentLL the word API is [not allowed](https://docs.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info) in the description, please remove it.

File: certified-connectors/Tesseron ASM Ticket/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,819 @@
...
+      }
+    },
+    "/CreateTicketPosition": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default"
+          },
+          "200": {
+            "description": "sucess",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Tesseron ASM Ticket/apiDefinition.swagger.json:283]: @developmentLL please make sure that descriptions and titles are grammatically correct and are free of typographical errors. You can refer to [English strings guidelines](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

File: certified-connectors/Tesseron ASM Ticket/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,819 @@
...
+                "description": "\"cache-control\"",
+                "type": "string"
+              },
+              "\"content-length\"": {
+                "description": "\"content-length\"",
+                "type": "string"
+              },
+              "\"content-security-policy\"": {
+                "description": "\"content-security-policy\"",
+                "type": "string"
```
Comment (v-EgorKozhadei) [inline certified-connectors/Tesseron ASM Ticket/apiDefinition.swagger.json:326]: @developmentLL could you please tell me what are these headers for? How are you going to use them?

File: certified-connectors/Tesseron ASM Ticket/apiDefinition.swagger.json [hunk 7]
```diff
@@ -0,0 +1,819 @@
...
+              },
+              "}": {
+                "description": "}",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "Create Ticket Position",
+        "description": "CreateTicketPosition",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Tesseron ASM Ticket/apiDefinition.swagger.json:372]: @developmentLL descriptions with CaMel case are considered non-translatable. Please make sure that descriptions and titles look like valid sentences. More information about English strings can be found in [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

File: certified-connectors/Tesseron ASM Ticket/apiDefinition.swagger.json [hunk 8]
```diff
@@ -0,0 +1,819 @@
...
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "title": "Tags"
+                  },
+                  "description": "Tags"
+                },
+                "ReleasedOption": {
+                  "type": "boolean",
+                  "description": "ReleasedOption",
```
Comment (v-EgorKozhadei) [inline certified-connectors/Tesseron ASM Ticket/apiDefinition.swagger.json:504]: @developmentLL please look through descriptions and make sure they meet [English strings standards](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

**Review by v-EgorKozhadei — COMMENTED** [review]
Hi @developmentLL,

Please check comments after the review. I'll also ask you to remove the icon from the PR, you'll be able to apply it later in ISV Studio.

Let us know if you have any questions.

File: certified-connectors/Tesseron ASM Ticket/readme.md (hunk 1 above)
Comment (developmentLL) [reply certified-connectors/Tesseron ASM Ticket/readme.md:4]: Mandatory headings and content was added

**Comment by v-EgorKozhadei** [comment]
Hi @developmentLL!

I can't see new changes for connector files. Please make sure that you have pushed the latest changes to your branch.

**Comment by developmentLL** [comment]
Hello @v-EgorKozhadei ,
are there additional steps for us before the review? I*ve commited my changes and deleted the Icon.

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
@developmentLL We have reviewed and approved your PR, please update your connector submission in the ISV Studio with the latest commit id you made in this PR. Thanks.

Docs - https://docs.microsoft.com/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio

**Pull request merged**